pkg/apiserver/authentication/internaloauth/expirationvalidator_test.go

```diff
@@ -0,0 +1,70 @@
+package internaloauth
```

Comment:
Can you do this as a move to preserve the history of this file.

Reply:
> Can you do this as a move to preserve the history of this file.

No.  The previous one part of a giant test file.  There is no previous file to move.